plugins/outputs/cloudwatchlogs/pusher.go

```diff
@@ -20,9 +20,9 @@ import (
 )
 
 const (
-	reqSizeLimit    = 1024 * 1024
-	reqEventsLimit  = 10000
-	logWarnInterval = 1 * 24 * time.Hour
+	reqSizeLimit     = 1024 * 1024
+	reqEventsLimit   = 10000
+	warnOldTimeStamp = 1 * 24 * time.Hour
```

Comment:
minor nit: Still think threshold should be somewhere in the name, but it isn't blocking.